exercises/grep/description.md

```diff
@@ -13,10 +13,9 @@ The `grep` command takes three arguments:
 3. One or more files in which to search for matching lines.
 
 Your task is to implement the `grep` function, which should read the contents
-of the specified files, find the lines that match the specified pattern
-and then output those lines as a single string. Note that the lines should
-be output in the order in which they were found, with the first matching line
-in the first file being output first.
+of the specified files and find the lines that match the specified pattern.
+The lines should be in the order in which they were found, with the first
+matching line in the first file being first.
```

Comment:
Check for clarity of this sentence. Perhaps it may not be clear that it's referring to the order of lines that the tested `grep` function returns (or other appropriate verb). Though I'm not quite sure what it might be misinterpreted to mean.

Reply:
Perhaps
> Your task is to implement the `grep` function: given a list of files, find all lines that match the specified pattern. Return the lines in the order they appear in the files. You'll also have to handle options, which control how matching is done, and how the results are to be reported.

Reply:
I like that @glennj  suggestion includes "return". In the current change it's not clear what the implementation is supposed to do with the lines it finds.

Reply:
I can agree. I will make this change now. I had to think about why this was different from https://github.com/exercism/problem-specifications/issues/1850 (hello-world). The difference there is that some tracks won't necessarily implement hello-world as a function, whereas I think with most tracks grep will be a function, so we are fine here.

Reply:
Agreed and implemented suggestion, with thanks.

I had to think about why this was different from https://github.com/exercism/problem-specifications/issues/1850 (`hello-world`). I now see that the difference is that some tracks won't necessarily implement `hello-world` as a function so we didn't want to use the word **return**, whereas I think with most tracks `grep` will be a function, so we are fine to use the word **return** for `grep`. I don't think there will be objections to that.